Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/StepDefinitions/TokensApiStepDefinitions.cs

```diff
@@ -141,11 +156,11 @@ public async Task WhenAPOSTRequestIsSentToTheTokensEndpointWith()
 
         if (_isAuthenticated)
         {
-            _createTokenResponse = await _sdk.Tokens.CreateToken(request);
+            _createTokenResponse = await _sdk!.Tokens.CreateToken(request);
         }
         else
         {
-            _createTokenResponse401 = await _sdk.Tokens.CreateTokenUnauthenticated(request);
+            _createTokenResponse401 = await _sdk!.Tokens.CreateTokenUnauthenticated(request);
```

Comment:
We might as well use the _createTokenResponse. No need for a separate one, as far as I can see

Reply:
They are of different type, it looks to me as it is possible, but it would take reworking the tests, sdks, handler etc...